buildSrc/src/main/kotlin/JsConfig.kt

```diff
@@ -34,7 +34,7 @@ fun Project.configureJs() {
 internal fun KotlinJsSubTargetDsl.useMochaForTests() {
     testTask {
         useMocha {
-            timeout = "10000"
+            timeout = "60000"
```

Comment:
I've increased timeout for JS tests here, as I have no idea from where the issue in JS engine comming :)

For some reason, the behavior in Js/WasmJs differs here - I can't understand if it's the issue in `ktor-network` implementation or in some expect-actual, or in some other thing...